pkg/envoy/envoy.go

```diff
@@ -205,10 +203,6 @@ func StartEnvoy(adminPort uint32, stateDir, logDir string, baseID uint64) *Envoy
 				close(crashCh)
 			}()
 
-			if logFile != nil {
-				logFile.Close()
-				logFile = nil
-			}
```

Comment:
`logger.Close()`?

Reply:
Lumbarjack is an `io.WriteCloser that writes to the specified filename.`

```Logger opens or creates the logfile on first Write. If the file exists and is less than MaxSize megabytes, lumberjack will open and append to that file. If the file exists and its size is >= MaxSize megabytes, the file is renamed by putting the current time in a timestamp in the name immediately before the file's extension (or the end of the filename if there's no extension). A new log file is then created using original filename.
```
I will need to see at what point we need to explicitly close it.

Reply:
Why not just `defer logger.Close()` just after creating it?

Reply:
Adding the `defer logger.Close()`

Reply:
done